src/blockchain/different-networks/ton-strategy.service.ts

```
@@ -0,0 +1,158 @@
+                        privateKey,
+                        to,
+                        amount,
+                        secondPrivateKey,
+                    );
+                    break;
+            }
+
+
+            return {
+                bundlerUrl: this.host,
+                signedTransaction: signedTransaction,
+                error: null,
+                transactionId: transactionId,
+            };
+        } catch (error) {
+            return {
+                bundlerUrl: this.host,
+                signedTransaction: null,
+                error: error.message,
+                transactionId: transactionId,
+            };
+        }
+    }
+
+    private async getSignedTransactionCoin(
+        privateKey: string,
+        to: string,
+        amount: Decimal,
+        secondPrivateKey: string,
+    ): Promise<SignedTonMessage> {
+        const { sender, feePayer } = this.recreateKeypairFromPreviouslyGeneratedSecretKey(privateKey, secondPrivateKey);
+        const source = WalletContractV4.create({
+            workchain: 0,
+            publicKey: sender.publicKey,
+        });
+        const testClient = new TonClient({ endpoint: this.host })
+        const walletContract = testClient.open(source);
+        const seqno = await walletContract.getSeqno();
+        const nanoAmount = DecimalsHelper.divide(amount, 10 ** this.chain.nativeCurrency.decimals);
+        const transactionIdentifier = uuidv4().replace(/-/g, '').substring(0, 55);
+        const transfer = internal({
+            to,
+            value: nanoAmount.toString(),
+            bounce: false,
+            body: transactionIdentifier,
+        });        
+
+        try {
+            // Step 1: Create unsigned message
+            const unsigned = source.createTransfer({
+                seqno,
+                secretKey: sender.secretKey,
+                sendMode: SendMode.PAY_GAS_SEPARATELY | SendMode.IGNORE_ERRORS,
```

ignore errors it not used ?